src/components/core/DpEditor/DpEditor.stories.tsx

```diff
@@ -0,0 +1,72 @@
+import type { Meta, StoryObj } from '@storybook/vue'
+import DpEditor from './DpEditor.vue'
+import DpModal from '../../DpModal/DpModal.vue'
+
+const meta: Meta<typeof DpEditor> = {
+    component: DpEditor,
+    title: 'Components/Editor',
+    render: (args) => ({
+        components: {
+            DpEditor
+        },
+        setup() {
+            return { args }
+        },
+        template: `<dp-editor v-bind="args"></dp-editor>`,
+    })
+}
+
+interface IDpEditor {
+    tusEndpoint: string
+    value: string
+    toolbarItems: object
+    routes: object
```

Comment:
Nitpick: alpnabetical sorting

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/567/commits/55c3ae63b443d85b1ea29de3ed94bf27d8193438